f600233d1b154d507b782f84604bcb12/update/sys_hub_action_type_definition_921e87a8978011101e0835dfe153afb4.xml

```diff
@@ -1583,10 +1583,12 @@ Request Body:
   
   
   var headers = signedRequest.getHeaderMap();
-  outputs.host_header = base_url;
+  outputs.base_url = base_url;
+  outputs.host_header = "api.atat.dev";
```

Comment:
Should this also become a system property?

Reply:
Ah thanks for catching that. I was troubleshooting and hard-coded that. Will revert that back to being parameterized.